examples/05_deduplication.py

```diff
@@ -107,17 +109,16 @@
 )
 
 ###############################################################################
-# .. _example_deduplication:
-#
 # Deduplication: suggest corrections of misspelled names
 # ------------------------------------------------------
 #
-# The |dd| function uses clustering based on
-# string similarities to group duplicated names
+# The |deduplicate| function uses clustering based on
```

Comment:
This is broken all over the example
![image](https://github.com/skrub-data/skrub/assets/208217/b2c937d1-ac68-4df8-9f06-159c7b474a5e)